techmatters/serverless
# Create control task for 'pre-survey' and clean up stale chat channel

## Description
- In Taskrouter callback in Twilio console, added `isChatCaptureControl` filter attribute to 'Survey' workflow 
- Created a task when the chat channel is captured, with the check on `isChatCaptureControl`
- Updated janitorListener to handle cleanup of this stale channel
- Move control task to `completed` assignment state

### Checklist



### Related Issues
Fixes # [1789](https://tech-matters.atlassian.net/browse/CHI-1789)

### Verification steps
- Deploy this branch to [Aselo Development](https://github.com/techmatters/serverless/actions/workflows/aselo_development.yml) via github actions
- In serverless console ([Twilio console](https://console.twilio.com/us1/develop/functions/services)), turn `Live logs` on. 
- Check that [dev web chat](https://tl-public-chat-as-dev.s3.amazonaws.com/as-chat-development.html)'s pre-survey is initiated with the lex programmed chatbot configured questions
- In serverless logs, ensure that this message appears: `Move control task to complete state`.
